tests/libraries/token/test_basic_token.py

```diff
@@ -0,0 +1,66 @@
+
+from ethereum.tools import tester
+from ethereum.tools.tester import TransactionFailed
+from utils import longToHexString, stringToBytes, captureFilteredLogs
+from pytest import fixture, raises
+
+
+def test_basic_token_transfer(localFixture, chain, mockToken):
+    assert mockToken.mint(tester.a1, 100)
+    assert mockToken.balanceOf(tester.a1) == 100
+    assert mockToken.balanceOf(tester.a2) == 0
+    with raises(AttributeError, message="can not call internal method"):
+        mockToken.internalTransfer(tester.a1, tester.a2, 100)
+
+    with raises(TransactionFailed, message="can not cause negative balance"):
+        mockToken.callInternalTransfer(tester.a1, tester.a2, 150)
+
+    with raises(TransactionFailed, message="can not cause negative balance"):
+        mockToken.callInternalTransfer(tester.a1, tester.a2, 150)
```

Comment:
This appears to be identical to the check immediately above.

Reply:
yes, dup. will remove